game/views.py

```
@@ -1015,4 +1027,108 @@ def password_reset_account_selection(request):
+""",
+                    from_email=settings.EMAIL_HOST_USER,
+                    recipient_list=[user.email],
+                    fail_silently=False,
+                )
+
+                messages.success(
+                    request,
+                    'Confirmation email sent to your registered email.'
+                )
+
+            except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)